src/app/get-involved/country-supporters.ts

```diff
@@ -0,0 +1,4 @@
+export interface CountrySupporters {
+  country: string;
+  count: number;
```

Comment:
As I mentioned, I would prefer having the id as part of the model. This data is supposed to be dynamic, so it could change in the future. I will be easier to make updates, deletes, gets if we have the id as part of the model and the document.

Reply:
Fixed